front/middleware/routeMiddleware.js

```diff
@@ -0,0 +1,6 @@
+export default function(context) {
```

Comment:
what its for? dont see it used anywhere

Reply:
I am doing some research on how we can show hide header based on page. According to design, at some pages header is present, but not on all pages. So, I was trying to implement route middleware to show hide header based on page and login status. For now I have done at page level through scss so will remove this.

Reply:
I've removed it.

Reply:
I've found that if I use scope, some styles are not being applied like i am not able to give color to icons. But On each page I have specified styles inside #page_name element. So that will not conflict on any other page, that i have made sure.